HomeKitBridge/accessories/contactSensor.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+class HAPAccessoryContactSensor extends HAPAccessoryBase
+{
+    public function __construct($data)
+    {
+        parent::__construct(
+            $data,
+            [
+                new HAPServiceAccessoryInformation(),
+                new HAPServiceContactSensor()
+            ]
+        );
+    }
+
+    public function readCharacteristicContactSensorState()
+    {
+        return GetValue($this->data['VariableID']);
+    }
+}
+
+class HAPAccessoryConfigurationContactSensor
+{
+    public static function getPosition()
+    {
+        return 110;
+    }
+
+    public static function getCaption()
+    {
+        return 'Contact Sensor';
+    }
+
+    public static function getColumns()
+    {
+        return [
+            [
+                'label' => 'ContactSensorState',
```

Comment:
Ich würde beides VariableID nennen, wie wir es auch in den anderen Accessories machen